internal/server/auth_expiry_test.go

```
@@ -0,0 +1,134 @@
+		expires:  time.Now().Add(-time.Second),
+	}
+
+	_, ok := h.getPKCEVerifier("state-1")
+	assert.False(t, ok, "expired verifier must not be accepted")
+	assert.NotContains(t, h.pkceVerifiers, "state-1", "expired verifier must be removed")
+}
+
+func TestStorePKCEVerifierSweepsExpiredEntries(t *testing.T) {
+	h := NewAuthHandler(nil, nil, false, "", nil)
+
+	for i := 0; i < 10; i++ {
```

golangci-lint (2.13.2)

[error] rangeint: for loop can be modernized using range over int

(modernize)